demisto_sdk/commands/common/tools.py

```diff
@@ -1195,6 +1195,9 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
                 'newEventFilters' in _dict and 'readyNewEventFilters' in _dict:
             return FileType.PRE_PROCESS_RULES
 
+        if 'allRead' in _dict and 'truncated' in _dict:
```

Comment:
Please also add to `find_type_by_path` if the file ends with json and is in lists dir as well

Reply:
Done